src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,57 @@
+package duke;
+
+/**
+ * Represents a Personal Assistant Chatbot that helps a person keeps track of various things.
+ *
+ * @author Sherman Ng Wei Sheng
+ */
+public class Duke {
+    private final Storage storage;
+    private final Ui ui;
```

Comment:
I like the use of final keyword here to indicate that storage and ui wouldnt be further modified.